plugins/org.eclipse.xtext.xbase.ide/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.6
 Bundle-ActivationPolicy: lazy
 Bundle-Localization: plugin
 Require-Bundle: org.eclipse.xtext.ide,
- org.eclipse.xtext.xbase,
+ org.eclipse.xtext.xbase;visibility:=reexport,
```

Comment:
Agreed. I was somehow concerned about re-exported re-exports, but those will be pulled into the mydsl.ide bundle via an explicit xbase dependency, too.